### PR TITLE
Reduce FV verbosity

### PR DESCRIPTION
**Pull request by fasaxc**
## Description

* Ignore infra logs during stop. (API server tends to log a few pages of "can't reach etcd", which is totally useless.)
* Log command output inline rather than buffer.  (Think the buffered approach was added before we understood we could route logrus to ginkgo's writer in a nice way...)
* Reduce k8s components log levels; I can't remember the last time we've hit a problem where those logs were useful.

## Todos
- [ ] Unit tests (full coverage)
- [ ] Integration tests (delete as appropriate) In plan/Not needed/Done
- [ ] Documentation
- [ ] Backport
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

File: fv/named_ports_test.go
Comment (neiljerram) [inline fv/named_ports_test.go:161]: I did not know you could call Info with a single object arg....

File: fv/utils/utils.go
Comment (neiljerram) [inline fv/utils/utils.go:94]: I think it's going to be a bit confusing having ` command=kubectl whatever` appended to the last line of the output.  Can we revert to logging the command on its own first?  It's not a big deal for me, but I think that would be better.

**Review by neiljerram — CHANGES_REQUESTED** [review]
Just one small thing...

File: fv/utils/utils.go
Comment (fasaxc) [reply fv/utils/utils.go:94]: Yes, was just thinking the same myself after trying this out for a bit :+1:

**Review by neiljerram — APPROVED** [review]
Looks good, thanks.

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and leave the branch after I've merged it.

**Pull request merged**
